en/releases/release-1.3.7.md

```diff
@@ -0,0 +1,25 @@
+---
+title: TiDB Operator 1.3.7 Release Notes
+---
+
+# TiDB Operator 1.3.7 Release Notes
+
+Release date: August 1, 2022
+
+TiDB Operator version: 1.3.7
+
+## New Features
+
+- Add the `suspendAction` field to suspend any component, it will delete the `StatefulSet` of the component ([#4640](https://github.com/pingcap/tidb-operator/pull/4640), [@KanShiori](https://github.com/KanShiori))
+
+## Improvement
+
+- Recreate the `StatefulSet` of a component after all PVCs scale-up, so that new PVCs use desired storage size ([#4620](https://github.com/pingcap/tidb-operator/pull/4620), [@KanShiori](https://github.com/KanShiori))
```

Comment:
```suggestion
- After all PVCs are scaled up, recreate the `StatefulSet` of a component so that new PVCs use the desired storage size ([#4620](https://github.com/pingcap/tidb-operator/pull/4620), [@KanShiori](https://github.com/KanShiori))
```